etc/profile-m-z/whois.profile

```diff
@@ -46,7 +46,7 @@ private
 private-bin bash,sh,whois
 private-cache
 private-dev
-private-etc jwhois.conf,services,whois.conf
+private-etc services,whois.conf
```

Comment:
That's not a typo, there is a whois implementation called jwhois.

Reply:
Thanks for catching this. TIL :+1: . PR to resolve is in.